pkg/features/kube_features.go

```diff
@@ -33,6 +33,8 @@ const (
 	//
 	// Enables partial admission.
 	PartialAdmission featuregate.Feature = "PartialAdmission"
+	// Enables cluster queue visibility.
+	ClusterQueueVisibility featuregate.Feature = "ClusterQueueVisibility"
```

Comment:
Could you expand this comment as similar to the `PartialAdmission`?

Reply:
added https://github.com/kubernetes-sigs/kueue/pull/1139/commits/0284c1110a8b841d28b80befee3fc8869cc6fa09